experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -84,6 +84,13 @@ import { Err, IgnoreMatcher, ParsedRequestOptions } from './internal-types';
 import { SYNTHETIC_BOT_NAMES, SYNTHETIC_TEST_NAMES } from './internal-types';
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 import forwardedParse = require('forwarded-parse');
+const STR_REDACTED = 'REDACTED';
```

Comment:
Add this and sensitiveParams as constants in internal-types.ts to keep same structure